doc/api/vm.md

```diff
@@ -304,29 +304,6 @@ console.log(util.inspect(sandbox));
 // { globalVar: 1024 }
 ```
 
-## vm.runInDebugContext(code)
-<!-- YAML
-added: v0.11.14
--->
```

Comment:
This should not be removed while it’s still usable. Instead, keep it in the docs add this to the metadata:
```yaml
deprecated: REPLACEME
changes:
  - version: REPLACEME
    pr-url: https://github.com/nodejs/node/pull/12243
    description: Calling this function emits a deprecation warning now.
```